docs/the-new-architecture/layout-measurements.md

```diff
@@ -0,0 +1,49 @@
+# Measuring the Layout
+
+Sometimes, you need to measure the current layout to apply some changes to the overall layout or to make decisions and call some specific logic.
+
+React Native provides some native methods to know what are the measurements of the views. The best way to invoke those methods is in a `useLayoutEffect` hook: this will give you the most recent values for those measurements and it will let you apply changes in the same frame when the measurements are computed. Typical code will look like this:
```

Comment:
```suggestion
React Native provides some native methods to know what are the measurements of the views. 

The best way to invoke those methods is in a `useLayoutEffect` hook: this will give you the most recent values for those measurements and it will let you apply changes in the same frame when the measurements are computed. 

Typical code will look like this:
```